### DISCUSSION TITLE
connecting to global Rustdesk unreliable at certain times of the day.

### DISCUSSION THREAD

**Discussion by jamiegau** [rustdesk/rustdesk#4524] [General]
### Bug Description

Trying to connect to a remote site will result in "Failed to connect via rendezvous server: Please try again"

I imagine this is because the global ID-Servers are too busy.  This happens to me often during the hours of 4-7pm  And sometimes other times randomly.
If I set the ID-server on both sides to to my own server, it will work fine.
Typically I still use my private server as the relay server.  But this makes no difference to the issues described above.

Two issues,
1.  get the global server to deal with busy times of the day better.  DO you need more (One in Australia)? AND
2.  Allow the user to set a primary ID-Server and allow it to fall back to the global server if it cannot find the ID.
This is an important major feature. as some people I help out don't want to dedicate a rustdesk connection to my ID-server for everything.  They want other agents the ability to connect as well as if it was on the global network too.

I consider this a major issue holding the adoption of this amazing software back.
It's a trivial change.

### How to Reproduce

This is likely location specific.

### Expected Behavior

Connect to remote site without issue.the 

### Operating system(s) on local side and remote side

all

### RustDesk Version(s) on local side and remote side

1.1.9 on all.

### Screenshots

NA

**Comment by rustdesk** [comment]
#1258
